docs/source/install/deb.rst

```diff
@@ -19,11 +19,10 @@ APT
 Pip
 '''
 
-The easiest way to install these is to use the requirements.txt file from the |st2| downloads server.  This is kept up to date for the version specified in the path.
+The easiest way to install these is to use the requirements.txt file from the |st2| repository.  This is kept up to date for the version specified in the path.
 
 ::
-
-    curl -q -k -O https://ops.stackstorm.net/releases/st2/0.9dev/requirements.txt
+    curl -q -k -O https://raw.githubusercontent.com/StackStorm/st2/master/requirements.txt
```

Comment:
0.9dev does not exist as a branch, and I assumed (incorrectly?) that TOT is 0.9dev.